drivers/neopixel/__init__.py

```diff
@@ -1,7 +1,11 @@
-# NeoPixel driver for MicroPython on ESP32
-# MIT license; Copyright (c) 2016 Damien P. George
+# NeoPixel driver for MicroPython
```

Comment:
Was there a reason to have this as `neopixel/__init__.py`, rather than just `neopixel/neopixel.py`?  Ie a single-file module rather than a whole package?  It would be slightly more efficient to import as a single-file, and also use a little less flash and RAM.

Reply:
I was considering making it so you could use the options to `include()` so that you could enable/disable features (exactly like what we do with aioble). But for now just neopixel.py is fine, we can always change it later if necessary. Updated.

Reply:
Great, thanks.  At this stage there is no difference to the user having it as a module or package, so module is simpler.